bundles/binding/org.openhab.binding.caldav-command/README.md

```diff
@@ -59,7 +61,9 @@ openhab.cfg
 
 ## Default item
 Since 1.9.0, the CalDAV Command binding also supports a default item which may optionally be added to openhab.cfg:
-<pre>caldavCommand:defaultItemOnBegin=&lt;item&gt;</pre>
+```
+caldavCommand:defaultItemOnBegin=&lt;item&gt;
```

Comment:
This doesn't render correctly this way.

Reply:
I am not very experienced with github. What exactly do you mean?